clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3951,7 +3958,7 @@ The capturing entity ``X`` can be one of the following:
       std::set<std::string_view> s;
     };
 
-- 'global', 'unknown' (without quotes).
+- `global`, `unknown`.
```

Comment:
We would give an error:
'lifetime_capture_by' attribute argument "global" is not a known function parameter; must be a function parameter, 'this', 'global' or 'unknown'.

Reply:
Sorry. I misunderstood your comment. I thought you were referring to a global within quotes here.
> ```cpp
> void captureInt(const int &i [[clang::lifetime_capture_by(global)]],
>                X &global);
> ```
This is already tested and was done as part of the syntax related changes for the annotation [PR](https://github.com/llvm/llvm-project/pull/115823), we would give:
`parameter cannot be named 'global' while using 'lifetime_capture_by(global)'`

Reply:
> Was this already discussed somewhere else?

This was discussed in https://github.com/llvm/llvm-project/pull/111499#discussion_r1836463450

> I think we should try to find an alternative for limiting the parameter name this way, since it's possible it won't be easy to change in some cases.

Changing parameter names should be relatively easy though. It is not part of the signature/mangled name so it should be relatively local change.
Other alternatives include 
- using parameter indices: But this reduces readability and is very brittle.
- `[[clang::lifetime_capture_by(__global)]]`: But this only reduces the likelyhood of name collision.

Reply:
Parameter names have enough constraints given local style guides and readability, so I'm not sure we want to add another constraint here.

`__global` makes sense to me.
I think it reduces the likelihood enough to make this a non-issue.
Probably worth doing in a separate PR.